### PR TITLE
fix(TESB-27788): Fix issue by packing takokit components along with t…

### PR DESCRIPTION
**Pull request by vdrokov**
…he microservice jar.

1) Place takokit jars into /BOOT-INF/classpath/TALEND-INF/repository/<mvn path> directory
2) Place plugins description into /TALEND-INF/plugins.properties file.

### CODE REVIEW

File: main/plugins/org.talend.designer.maven/resources/templates/maven_user_settings.xml [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
 
+	<mirrors>
+		<mirror>
+			<id>official</id>
+			<name>repo.maven.apache.org</name>
+			<url>https://repo.maven.apache.org/maven2</url>
+			<mirrorOf>central</mirrorOf>
+		</mirror>
+	</mirrors>
+
```
Comment (yanbasic) [inline main/plugins/org.talend.designer.maven/resources/templates/maven_user_settings.xml]: This part is not related to this issue, please do not commit it.

File: main/plugins/org.talend.libraries.jackson/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -35,7 +35,7 @@ Require-Bundle: org.apache.commons.lang,
  org.apache.commons.beanutils,
  jackson-core-asl;bundle-version="1.9.13";visibility:=reexport,
  jackson-mapper-asl;bundle-version="1.9.13";visibility:=reexport,
- com.fasterxml.jackson.core.jackson-annotations;bundle-version="2.9.0";visibility:=reexport,
+ com.fasterxml.jackson.core.jackson-annotations;bundle-version="2.9.5";visibility:=reexport,
```
Comment (yanbasic) [inline main/plugins/org.talend.libraries.jackson/META-INF/MANIFEST.MF]: It's not idealy to change the jackson library in the maintenance branch, also some potential side effects need to check with other plugins, and we also need to provide this version into repository in studio before to upgraeding the versions.

File: main/plugins/org.talend.designer.maven/resources/templates/maven_user_settings.xml (hunk 1 above)
Comment (vdrokov) [reply main/plugins/org.talend.designer.maven/resources/templates/maven_user_settings.xml]: Will not work without it

Comment (vdrokov) [reply main/plugins/org.talend.designer.maven/resources/templates/maven_user_settings.xml]: Because we need to build a microservice and maven issue will not allow to make a build

File: main/plugins/org.talend.libraries.jackson/META-INF/MANIFEST.MF (hunk 2 above)
Comment (vdrokov) [reply main/plugins/org.talend.libraries.jackson/META-INF/MANIFEST.MF]: Ideally there will not be any side effects because we align the version of the library with version of the other libraries of the same family, this will make it work more safe

**Comment by dsergent** [comment]
Will be handle in scope of a new feature project for supporting TaCoKit components in OSGi Builds.
